lms/static/styles/components/_LMSGrader.scss

```diff
@@ -1,39 +1,55 @@
 .LMSGrader {
   display: flex;
   flex-direction: column;
-  font-size: 18px;
+  font-size: 16px;
   height: 100%;
   width: 100%;
   position: fixed;
-  color: white;
-  .LMSGrader__course-assignment {
-    h2, h3 {
-      margin-left: 20px;
-    }
-  }
+  color: $grey-6;
 }
 
 .LMSGrader__grading-components {
-  background-color: $grey-7;
-  padding: 0;
+  background-color: white;
+  padding: 15px 15px;
   margin: 0;
   list-style: none;
   display: flex;
 }
 
-.LMSGrader__assignment {
-  flex-grow: 1;
-}
-
-.LMSGrader__student-index {
+.LMSGrader__grading-components--label-wrapper {
   display: flex;
   flex-flow: column;
   justify-content: space-around;
-  span {
-    margin: 10px;
-  }
+  margin: 10px;
+  font-weight: 500;
+}
+
+.LMSGrader__assignment {
+  flex-grow: 1;
 }
 
 .LMSGrader__student-picker {
   display: flex;
+  align-items: center;
+  margin-right: 20px;
+}
+
+
+.LMSGrader__loading-backdrop {
+  background-color: white;
+  bottom: 0;
```

Comment:
Wanted to sanity check that we really need all four (top, bottom, right, left) positioning values here.